src/deps/uws.zig

```diff
@@ -40,6 +40,129 @@ pub fn NewSocketHandler(comptime ssl: bool) type {
             return us_socket_timeout(comptime ssl_int, this.socket, seconds);
         }
 
+        pub fn open(this: ThisSocket, is_client: bool) void {
+            _ = us_socket_open(comptime ssl_int, this.socket, @intFromBool(is_client), null, 0);
+        }
+
+        // Note: this assumes that the socket is non-TLS and will be adopted and wrapped with a new TLS context
+        // context ext will not be copied to the new context, new context will contain us_wrapped_socket_context_t on ext
+        pub fn wrapTLS(
+            this: ThisSocket,
+            options: us_bun_socket_context_options_t,
+            socket_ext_size: i32,
+            comptime deref: bool,
+            comptime ContextType: type,
+            comptime Fields: anytype,
+        ) ?NewSocketHandler(true) {
+            const @"type" = comptime if (@TypeOf(Fields) != type) @TypeOf(Fields) else Fields;
```

Comment:
usually the convention is to have `Type` be capitalized

Reply:
fix in https://github.com/oven-sh/bun/pull/3457/commits/68b783e0bbb8c6721f291fbbfae114197b550491